### ISSUE TITLE
Problems with url

### ISSUE THREAD

**Issue by falolaf** [rNeomy/tab-suspender#16]
Hello,

There seems that tab-suspender has problems with this link:
http://www.idg.se/2.33206/dagens-dilbert-pa-idgse/2017-03-22

I can see an error message after it gets suspended:
Firefox can't find the file on moz-extension://222de203-a814-4606-be07-af086f7b6e99/data/suspend/index.html?title=Dagens%20Dilbert%20-%202017-03-22%20-%20IDG.se&url=http%3A//www.idg.se/2.33206/dagens-dilbert-pa-idgse/2017-03-22&favicon=http%3A//www.idg.se/polopoly_fs/2.109.1489654539!/favicon.ico

No other pages I use have this issue. 

/Anders

**Comment by rNeomy** [comment]
I am not able to reproduce this on version 0.1.8. Please wait for this version gets approved and reopen if you still have the issue.

**Comment by falolaf** [comment]
I have now tested with 0.1.8 and I still see the same behaviour. Same happens with: https://www.idg.se/

FF 55.0.2
Windows 7

**Comment by yfdyh000** [comment]
I see this issue in 0.1.8 with Firefox 56.

**Comment by pabuisson** [comment]
I got this issue twice this afternoon while testing the addon, with two pinned tab. It had worked previously for two other pinned tabs.

Version 0.1.8
FF Nightly 57 2017-08-29

<img width="1274" alt="capture d ecran 2017-08-29 a 19 08 38" src="https://user-images.githubusercontent.com/769918/29842957-b5d1f7fa-8d0a-11e7-96d0-82f0887433b7.png">

When it happened, it made the tab try to load something continuously, therefore on my machine CPU was stuck around 100% almost all the time. I've had to close the tab each time it happened to get my CPU back to normal.
